tests_peerscout_api/end2end_tests/end2end_test.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import requests
+
+LOGGER = logging.getLogger(__name__)
+
+# to run the test locally use the url below:
+# URL = 'http://localhost:8090/api/peerscout'
+URL = 'http://peerscout-api:8080/api/peerscout'
```

Comment:
minor:
Just to explain what I meant...
I suggested environment variable so that it can be passed in accordingly. (e.g. via docker compose we could pass in the `http://peerscout-api:8080/api/peerscout` whereas the default could be `http://localhost:8090/api/peerscout`.
Usually it is not good to need to change code in order to run things (as it is easy to miss when committing and also not obvious one needs to look at this file).
Now the end to end test are functional via Docker but not a virtual env.
And that is in itself also okay. Therefore making it work for a virtual env as well is more like a nice to have.

Reply:
Could you check it please?

Reply:
Looking good, thank you